contracts/stake-cw20-rewards/src/contract.rs

```diff
@@ -161,5 +163,486 @@ pub fn query_config(deps: Deps) -> StdResult<GetConfigResponse> {
     })
 }
 
+pub fn query_rewards(deps: Deps, env: Env) -> StdResult<GetPendingRewardsResponse> {
+    let config = CONFIG.load(deps.storage)?;
+    let last_claim = LAST_CLAIM.load(deps.storage).unwrap_or(config.start_block);
+    let blocks = Uint128::from(min(&env.block.height, &config.end_block) - last_claim);
+    let pending_rewards = if config.funded {
+        blocks
+            .checked_mul(config.payment_per_block)
+            .map_err(StdError::overflow)?
+    } else {
+        Uint128::zero()
+    };
```

Comment:
Does this contract have a `payment_block_delta` like the other one? If so I don't think this is quite accurate. If not, we should discuss if they should be consistent. https://github.com/DA0-DA0/dao-contracts/pull/170#discussion_r802210734

Reply:
This contract doesn't have a delta like external, as rewards are paid every block so I believe the maths here is correct in line with @ben2x4 's description in PR #164 but correct me if I'm misunderstanding.